# Status by county

In [1]:
import pandas
import geopandas
from create_absentee_ballots_csv import create_absentee_ballots_df

df = create_absentee_ballots_df()
gdf = geopandas.read_file("./shapefiles/GA_precincts16.shp")

In [2]:
errors = df.groupby("COUNTY")["PRES16D"].sum() - gdf["PRES16D"].astype(int).groupby(gdf["CTYNAME"]).sum()

errors[errors != 0]

COUNTY
Bacon               89
Chatham         -34984
Chattahoochee     -567
Columbia          -666
Douglas           -408
Morgan             298
Name: PRES16D, dtype: int64

## The problem counties

### Counties with bad match scores:

- Bacon
- Bleckley ✅
- Chatham
- Chattahoochee
- Clarke
- Crawford ✅
- Rockdale
- Spalding ⚠️
- Taylor

### Counties with different numbers of precincts:

- Bibb 1 ✅
- Chattooga 1
- Columbia 3
- Crawford 3 ✅
- DeKalb 3
- Effingham 2
- Fayette 2
- Forsyth 4
- Fulton 1
- Gwinnett 1
- Henry 1 ✅
- Lowndes 2
- Muscogee 2
- Newton 2
- Pickens 1 ✅

### Counties with the wrong vote totals

- Bacon              89
- Barrow ✅ (added spaces to `txt` file)
- Chattahoochee    -567
- Chatham ✅ (added spaces to `txt` file)
- Columbia         -666
- DeKalb ✅ (added spaces to `txt` file)
- Douglas          -408
- Morgan            298